packages/typegpu/tests/computePipeline.test.ts

```diff
@@ -58,16 +58,21 @@ describe('TgpuComputePipeline', () => {
       );
   });
 
-  it('allows to omit input in entry function shell', () => {
-    expectTypeOf(
-      tgpu['~unstable'].computeFn({ in: {}, workgroupSize: [1] }),
-      // biome-ignore lint/complexity/noBannedTypes: it's fine
-    ).toEqualTypeOf<TgpuComputeFnShell<{}>>();
+  it('is resolvable', ({ root }) => {
+    const entryFn = tgpu['~unstable']
+      .computeFn({ workgroupSize: [32] })(() => {
+        // do something
+      })
+      .$name('main');
```

Comment:
$name is not required anymore, we could remove it in this case 👀 
```suggestion
    const main = tgpu['~unstable']
      .computeFn({ workgroupSize: [32] })(() => {
        // do something
      });
```